index.js

```diff
@@ -124,6 +124,7 @@ async function scrapeData() {
     else {
       counties++;
     }
+    location["active"] = transform.getActiveFromLocation(location)
```

Comment:
@osvyg just realized this should probably check if `location.active` is already defined... perhaps the scraper or datasource already has it... Should we use that value if so?

Reply:
True, ill do a check. I'll create a new PR

Reply:
Yeah, I think we should trust what the active data is coming back from the scraper. If none then we calculate it